docs/UserGuide.md

```diff
@@ -579,7 +581,9 @@ $ settlePerson
 
 Invalid command format! 
 settlePerson: Settle any outstanding balance with another person.
-If a timestamp is provided, the balance at the instant before the timestamp is used. Else, the balance at the instant before the current system time is used.
+If a timestamp is provided, the balance at the instant before 
+the timestamp is used. Else, the balance at the instant before 
+the current system time is used.
```

Comment:
Before and during actually. We include the minute of that transaction as well.

Reply:
So for example if there is an existing transaction at 12:00 and the timestamp for settlePerson is 12:00, that transaction at 12:00 is included

Reply:
I think it should be added in case testers call it a boundary value bug

Reply:
Yes, that is what the app is doing from what I was testing

Reply:
Oh wait this is the error message from the app oops